executor/split.go

```diff
@@ -352,3 +357,158 @@ func (e *SplitTableRegionExec) getSplitTableKeys() ([][]byte, error) {
 	}
 	return keys, nil
 }
+
+// RegionMeta contains a region's peer detail
+type regionMeta struct {
+	region     *metapb.Region
+	leaderID   uint64
+	start      string
+	end        string
+	scattering bool
+}
+
+func getPhysicalTableRegions(physicalTableID int64, tableInfo *model.TableInfo, tikvStore tikv.Storage, s splitableStore) ([]regionMeta, error) {
+	uniqueRegionMap := make(map[uint64]struct{})
```

Comment:
If multiple tables are located in one region, maybe we need to move `uniqueRegionMap` as a function parameter, otherwise, it can't deduplication?

Reply:
Great, thanks.